flask_rebar/rebar.py

```diff
@@ -459,9 +462,18 @@ def add_handler(
         :param Type[USE_DEFAULT]|None|str mimetype:
             Content-Type header to add to the response schema
         """
-        if isinstance(response_body_schema, marshmallow.Schema):
+        if isinstance(response_body_schema, marshmallow.Schema) or isclass(
```

Comment:
```suggestion
        if response_body_schema is not None:
```
(Consolidate the further checking and transformation of what was passed in to the new normalize function)

Reply:
I think we still need this (or a similar) bit of extra checking as the user could have passed either a schema by itself or a dictionary of {response: schema} items; if they passed a schema (or schema class) it still needs to get wrapped in the dictionary.

Reply:
(Oh duh, sorry! Have a followup suggestion, but I'll leave it against the latest version)